spec/adapters/googlecharts_spec.rb

```diff
@@ -88,4 +103,17 @@
       expect{Daru::View::Table.new("daru")}.to raise_error(ArgumentError) 
     end
   end
+
+  describe "Export html file of the chart" do
+    it "export_html_file method should write a file" do
+      allow(File).to receive(:write)
+      area_chart_chart.
+        export_html_file('./plot.html','f75b3117-7cb0-4451-add4-e0006cb10876')
+      path = File.expand_path('../../plot.html', __dir__)
```

Comment:
I don't think that it is good to test whole html code here. [As I said](https://github.com/SciRuby/daru-view/pull/84#issuecomment-371818560)  
we must test all related methods, so it is better to start test from the lower to upper level methods. Test for `initial_script ` , `chart_script ` i.e. `#to_html`, `#generate_body` ,`#generate_html` methods .

I think we must test for specific keys (in this case area chart, options, data, etc in javascript code) . Is it good to test whole file at once ? Ping @zverok @v0dro

Reply:
As I was busy with the proposal I was unable to work on this. 
I will test for the methods separately but I still think that testing the specific keys would be bug-prone. For instance, to_html method will return the rendered html in string format in ruby. So, extracting the chart, options and data in javascript code might be a diificult task.

Reply:
Because for extracting the `chart`, `options` and `data` from the JavaScript code, I think I will have to search for this text in the string returned. Is there any way to directly get those keys from the JS code without implementing a search?

Reply:
I believe pattern matching will be useful. You can explore how it is tested in `google_visualr`, `lazy_high_charts` or other gems like `formtastic` etc.